swift-pipeline

```diff
@@ -285,14 +222,21 @@ if __name__ == "__main__":
             print_if_debug(
                 f"Using registration functions contained in {registration_filename}"
             )
-
+            
+        global_mask_tag = config.auto_plotter_global_mask
```

Comment:
Can we add a small comment above to explain what this global mask tag is for? (it is not clear by only looking at the code)

Reply:
Thanks!